### PR TITLE
Added "Offset Selection" action

### PR DESCRIPTION
**Pull request by zWolfrost**
Hi, i have added the GUI option for offsetting the selection in the edit tab, finally closing my issue (#661).
Similarly to last time, I have tried my best to respect the project flow, but feel free to point out if i did something wrong.
I will add an icon for the action in a next commit if i manage to find one, but for now i will just use the same icon of the "invert selection" option.

### CODE REVIEW

**Comment by zWolfrost** [comment]
I have noticed that the code formatting check for Ubuntu wasn't successful, but that should be an unrelated problem and not caused by my pull request. Shouldn't be too bad, the build checks all work anyway.

File: Pinta/Actions/Edit/OffsetSelectionAction.cs [hunk 1]
```diff
@@ -0,0 +1,72 @@
...
+		dialog.OnResponse += (_, args) => {
+			if (args.ResponseId == (int) Gtk.ResponseType.Ok)
+			{
+				PintaCore.Tools.Commit ();
+
+				Document document = PintaCore.Workspace.ActiveDocument;
+
+				document.Layers.ToolLayer.Clear ();
+
+				SelectionHistoryItem historyItem = new SelectionHistoryItem (Resources.Icons.EditSelectionOffset, Translations.GetString ("Offset Selection"));
```
Comment (cameronwhite) [inline Pinta/Actions/Edit/OffsetSelectionAction.cs:58]: nitpick - could use `var historyItem = ...` here

**Review by cameronwhite — COMMENTED** [review]
This looks very good, thank you!

I think the formatting issue might be from having incorrect line endings?

**Comment by zWolfrost** [comment]
I have done the little change. About the formatting issue, all the errors are concentrated in "Pinta.csproj" and i didn't change it in the slightest so i feel like it should need another PR.

**Comment by zWolfrost** [comment]
There you go. So in the end it was caused by my PR, my bad. For future reference, run the command "dotnet format --no-restore --exclude-diagnostics CA1416" to let .NET format your code before submitting.

**Comment by cameronwhite** [comment]
Looks good, thanks for working on this!

**Pull request merged**